packages/space-opera/src/reducers.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the 'License');
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an 'AS IS' BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+// TODO: Refactor reducers to follow best practices, then combine them all here.
```

Comment:
Oh, perhaps this addresses my comments above? Will a bunch of those short files end up combined here instead?

Reply:
This file is for importing and combining the reducers into one using the combineReducers method from redux. In general, the top level isn't supposed to contain any reducers.